.changeset/wicked-crews-hammer.md

```diff
@@ -0,0 +1,40 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-api": minor
+---
+
+Add possibility to add application-defined configurations for permissions
+
+1. Define in API: Add type to Permission-Interface:
+
+```diff
+declare module "@comet/cms-api" {
+    interface Permission {
+-        news: string
++        news: {
++           commentsEdit: boolean;
```

Comment:
I know this is only a made up example, but won't you use a simple string permission "newsCommentsEdit" in pratice?

Reply:
> Nevertheless, the real advantage of this approach is to be able to add custom controls (example: a dropdown which allows you to restrict editing only a certain news-category).

Should we change the example in Demo so it reflects this use case? Or is this overkill?